app/controllers/state_file/questions/sms_terms_controller.rb

```diff
@@ -1,6 +1,9 @@
 module StateFile
   module Questions
     class SmsTermsController < QuestionsController
+      def self.show?(intake)
+        intake.phone_number?
```

Comment:
I think you'll also want to update how the terms and condition controller shows if this one is skipped

Reply:
don't we always want Terms and Conditions to show?